src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -1254,6 +1264,21 @@ bool clang_c_convertert::get_expr(const clang::Stmt &stmt, exprt &new_expr)
     if(get_decl_ref(dcl, new_expr))
       return true;
 
+    // if type is lvalue ref, convert to dereference subtree
+    if(is_lvalue_reference(dcl))
+    {
+      // get type and make a dereference subtree
```

Comment:
mark the irep tree and move this conversion to adjuster

Reply:
Tried 4a3c7d4fd4f92f02ac6a7ab28d5f91f8385df8e7 but it's making the adjust code structure too messy. 
Need to figure out a better way to adjust DeclRefExpr.